web/src/main/webapp/xslt/ui-metadata/form-builder.xsl

```diff
@@ -106,19 +106,16 @@
 
     <xsl:variable name="isoType" select="if (../@gco:isoType) then ../@gco:isoType else ''"/>
 
+    <xsl:variable name="elementCondition" select="gn-fn-metadata:getLabel($schema, name(), $labels, name(..),$isoType, $xpath)/condition" />
+
     <!-- Required status is defined in parent element for
     some profiles like ISO19139. If not set, the element
     editing information is used.
     In view mode, always set to false.
     -->
     <xsl:variable name="isRequired" as="xs:boolean">
       <xsl:choose>
-        <xsl:when
-          test="($parentEditInfo and $parentEditInfo/@min = 1 and $parentEditInfo/@max = 1) or
-          (not($parentEditInfo) and $editInfo and $editInfo/@min = 1 and $editInfo/@max = 1)">
-          <xsl:value-of select="true()"/>
-        </xsl:when>
-        <xsl:when test="gn-fn-metadata:getLabel($schema, name(), $labels, name(..),$isoType, $xpath)/condition = 'mandatory'">
+        <xsl:when test="$elementCondition = 'mandatory'">
```

Comment:
It seems like this will remove all dtd checks - my recommendation was to check for optional which would override the dtd check.

i.e. 

```
        <xsl:when test="$elementCondition = 'optional'">
		   <xsl:value-of select="false()"/>
        </xsl:when>
        <xsl:when
          test="($parentEditInfo and $parentEditInfo/@min = 1 and $parentEditInfo/@max = 1) or
          (not($parentEditInfo) and $editInfo and $editInfo/@min = 1 and $editInfo/@max = 1)">
          <xsl:value-of select="true()"/>
        </xsl:when>
        <xsl:when test="$elementCondition = 'mandatory'">
				   <xsl:value-of select="true()"/>
        </xsl:when>
```